jina/hubble/hubio.py

```diff
@@ -391,51 +391,53 @@ def push(self) -> None:
                     dict_data=form_data,
                     headers=req_header,
                     stream=True,
-                    method=method,
+                    method='post',
                 )
 
-                result = None
+                image = None
                 for stream_line in resp.iter_lines():
                     stream_msg = json.loads(stream_line)
 
-                    if 'stream' in stream_msg:
+                    t = stream_msg.get('type')
+                    subject = stream_msg.get('subject')
+                    if t == 'error':
+                        msg = stream_msg.get('message')
+                        payload = stream_msg.get('payload')
+                        if payload and isinstance(payload, dict):
+                            msg = payload.get('message')
+
+                        if 'Process(docker) exited on non-zero code' in msg:
```

Comment:
This is not really a good way to decide anything

Reply:
@delgermurun @nomagick Could we move this detection to server side and also using status code instead of message to detect the error somehow?